README.md

```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
 sudo docker build -t analythium/shinyproxy-demo .
 ```
 
-Test locally
+
+
+## Develop and test locally
+```bash
+docker-compose up --build
+```
+then visit `http://localhost:4000`.  To develop, simply make changes to your Shiny app and then reload the page!
+
+
+
+
+## Production build, test, and deploy
```

Comment:
we don't have to include deployment docs at all if you don't want.  i just found it helpful for myself.